src/client/scss/styles.scss

```
@@ -1,8 +1,178 @@
-@import "../scss/_variable";
 @import "../scss/_reset";
+@import "../scss/__variable";
+@import "../scss/__mixin";
 
-body {
-  background: $red;
-}
 
 
+.create-form {
+  width: 600px;
+  margin: 0 auto;
+  margin-top: 50px;
+
+  .input-title {
+    display: inline-block;
+    border: none;
+    width: 600px;
+    height: 30px; 
+    font-size: $font-20;
+    font-weight: bold;
+    margin-bottom: 20px;
+  }
+
+  .select-box { 
+    width: 220px;
+    height: 35px;
+    background-color: #EBEBEB;
+    color: $dark-gray;
+    font-size: $font-14;
+    border-radius: 10px;
+    border: none;
+    text-align: left; 
+    i{
```

어떤 태그는 띄어쓰기를 하고 어떤 태그는 하지 않는데 통일성에 맞게 해주시면 좋을거 같습니다.
```suggestion
    i {
```